pkgs/gcc/gcc.yaml

```diff
@@ -22,6 +22,14 @@ build_stages:
       patch -up1 < _hashdist/rpath.patch
       sed -i "s|@@ARTIFACT@@|${ARTIFACT}|g" gcc/config/i386/gnu-user.h gcc/config/i386/gnu-user64.h
 
+  - when: platform == 'Darwin'
+    name: fix_object_h
+    before: configure
+    handler: bash
+    bash: |
+      mkdir -p "$ARTIFACT/include/dispatch"
+      sed 's+typedef void (\^dispatch_block_t)(void)+typedef void* dispatch_block_t+' /usr/include/dispatch/object.h > "$ARTIFACT/include/dispatch/object.h"
```

Comment:
I think I should add a check that /usr/include/dispatch/object.h exists in the first place, and then also do this fix for OS X 10.10 only.